substrate/frame/broker/src/benchmarking.rs

```diff
@@ -852,6 +852,27 @@ mod benches {
 		}
 	}
 
+	#[benchmark]
+	fn do_tick_base() -> Result<(), BenchmarkError> {
+		setup_and_start_sale::<T>()?;
+
+		advance_to::<T>(5);
+
+		let mut status = Status::<T>::get().ok_or(BenchmarkError::Weightless)?;
+		status.last_committed_timeslice = 3;
+		Status::<T>::put(&status);
+
+		#[block]
+		{
+			Broker::<T>::do_tick();
+		}
+
+		let updated_status = Status::<T>::get().ok_or(BenchmarkError::Weightless)?;
```

Comment:
```suggestion
		let updated_status = Status::<T>::get().unwrap();
```
Cannot be weightless after it executed.